README.md

```diff
@@ -21,6 +21,20 @@ information](https://github.com/G-Node/nix/wiki)
 
 The current implementations store the actual data using the [HDF5](http://www.hdfgroup.org/) file format as a storage backend.
 
+Documentation --- where to find which information?
+--------------------------------------------------
+
+Project documentation is split up into three parts:
+1. Technical information: Please visit the GitHub repository [Wiki](https://github.com/G-Node/nix/wiki).
+2. General Introduction and tutorial: Visit the project pages on [ReadTheDocs](https://nixio.readthedocs.io/en/latest/)
+3. API documentation.
```

Comment:
Where is that? ;)

Reply:
I think the API docs aren't available right now, are they?  Figured we can add the link when we're done sorting.  Maybe add a note (or issue) about it.

Reply:
yes, I want to see, if the doxygen sphinx interaction works on read the docs. When it does the link will be set

Reply:
at the moment the API doc is actually nowhere